Library/Homebrew/brew.sh

```diff
@@ -402,15 +409,17 @@ Minimum required version: ${HOMEBREW_MINIMUM_CURL_VERSION}
 Your curl version: ${curl_name_and_version##* }
```

Comment:
`Please update your system curl.` above should only refer to `system curl` if `HOMEBREW_CURL_PATH` wasn't set.